packages/swirl-components/src/components/swirl-date-picker/swirl-date-picker.css

```diff
@@ -207,6 +207,10 @@
   }
 }
 
+.date-picker__date--overflowing {
```

Comment:
I moved this here to solve a styling problem for a selected date outside the range of current calendar.

**Before:**
<img width="274" alt="Screenshot 2024-02-22 at 18 00 06" src="https://github.com/getflip/swirl/assets/2084270/ceb42a7c-72c0-40b2-94bd-1debe6996f46">

**After:**
<img width="272" alt="Screenshot 2024-02-22 at 18 07 42" src="https://github.com/getflip/swirl/assets/2084270/96ea9d32-65cb-4d1d-8b6e-e84186fb5d05">